backend/requirements/default.txt

```diff
@@ -21,8 +21,8 @@ httpx==0.23.3
 httpx-oauth==0.11.2
 huggingface-hub==0.16.4
 jira==3.5.1
-langchain==0.0.325
-litellm==0.12.5
+langchain==0.0.340
+litellm==1.7.1
```

Comment:
want to try some of the new gpt-4 turbo stuff soon anyhow, might as well upgrade these